system/services/formbuilder/FormBuilderFilterService.cfc

```diff
@@ -570,6 +570,112 @@ component {
 		return response;
 	}
 
+	public boolean function evaluateUserHasSubmittedForm(
+		  required string userId
+		, required array extraFilters
+	) {
+		var websiteUserDao = $getPresideObject( "website_user" );
+
+		var filter = {
+			id = arguments.userId
+	  	}
+
+		result.records = websiteUserDao.selectData(
```

Comment:
A cleaner and more efficient approach here is to use `dataExists()` rather than `selectData()`. This will return a `boolean` and select less data from the DB. i.e. this entire function call can be just:

```cfc
return $getPresideObject( "website_user" ).dataExists(
    filter       = { id=arguments.userId }
  , extraFilters = arguments.extraFilters
);
```